javascript/src/test/components/Nav/AppNavbar.test.js

```diff
@@ -39,8 +39,6 @@ describe("AppNavbar tests", () => {
       </Router>
     );
 
-    const userInfoLink = getByText(/Profile/);
-    expect(userInfoLink.href).toMatch("/profile");
```

Comment:
It looks like this test was originally just to check for the presence of the Profile tab that you removed. Since that feature was removed entirely, you can remove this test as well, instead of just taking out the assertions.

Reply:
Just fixed this, thanks for the feedback!